<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddVisitDialog</class>
 <widget class="QDialog" name="AddVisitDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Додати відвідування</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>382</width>
     <height>282</height>
    </rect>
   </property>
   <property name="title">
    <string>Додати Відвідування</string>
   </property>
   <widget class="QLabel" name="studentIdLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>ID студента:</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Дата:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="diagnosisLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Діагноз:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="diagnosisEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="doctorEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="doctorLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Лікар:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="studentNameEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ім'я студента:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="studentIdEdit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>261</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>341</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="submitButton">
      <property name="text">
       <string>Додати</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="revertButton">
      <property name="text">
       <string>Очистити</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeButton">
      <property name="text">
       <string>Закрити</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
